srcalc/src/test/java/gov/va/med/srcalc/domain/variable/DiscreteNumericalVariableTest.java

```diff
@@ -31,15 +31,18 @@ public final void testCategoryOrdering()
                 new NumericalRange(4.0f, true, 5.0f, false), new MultiSelectOption("two"));
         final Category cat3 = new Category(
                 new NumericalRange(5.0f, true, 6.0f, false), new MultiSelectOption("three"));
+        // WNL should always be at the start of the list after sorting
+        final Category cat4 = new Category(
+        		new NumericalRange(4.0f, true, 4.5f, false), new MultiSelectOption("WNL"));
```

Comment:
Since you're making a change anyway, at least add a test that cat4.compareTo(cat4) == 0. The other place I tested the full compareTo() contract is in NumericalRangeTest if you want to implement that test helper method and use it from both places.